wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/button/CommandGenericHID.java

```diff
@@ -212,7 +222,9 @@ public Trigger axisLessThan(int axis, double threshold) {
    *     threshold.
    */
   public Trigger axisLessThan(int axis, double threshold, EventLoop loop) {
-    return m_hid.axisLessThan(axis, threshold, loop).castTo(Trigger::new);
+    var cache = m_axisCache.computeIfAbsent(loop, k -> new HashMap<>());
+    return cache.computeIfAbsent(
+        new Pair<>(axis, threshold), k -> new Trigger(loop, () -> getRawAxis(axis) < threshold));
```

Comment:
```suggestion
        new Pair<>(axis, threshold), k -> new Trigger(loop, () -> getRawAxis(axis) < threshold));
```
Nit: `Pair.of(axis, threshold)` reads cleaner.